### ISSUE TITLE
Enhance install script description for apache2 users about required modules

### ISSUE THREAD

**Issue by byPARSE** [zammad/zammad#5415]
As an admin that installs apache2 before Zammad I want to get a hint about required modules.

Enhance the following message:
![Image](https://github.com/user-attachments/assets/de5dadea-b32a-474f-809d-60e866dbb63c)

Add something like `Ensure required modules are enabled if apache does not start.`

<details>
  <summary>More background information</summary>

Preinstalled apache2. When installing zammad from scratch apache2 won't start after installation of zammad.
apache2 - Error:

![Image](https://github.com/user-attachments/assets/11b978b1-35cf-440e-96fb-e59046a26532)

"journalctl -xeu apache2.service" shows the following error:

![Image](https://github.com/user-attachments/assets/57b88b68-2202-4f36-8cba-e28ec8e50df0)

The line 45 is this line:

![Image](https://github.com/user-attachments/assets/789731e7-358e-4efe-b4f4-281e52efbf24)

Commenting this line out fixes the issue! apache2 restarts like a charm.

### Expected behaviour

After installing zammad from scratch apache2 has to start normally out of the box.

### Steps to reproduce the behaviour

First install apache2 in debian. After this, install zammad via package installation. The package-installer will give you the described error direct when installation has finished. I tried this in three completely different debian-instances. All of them has the same "error".
It's only the one line 45 in the apache2-template(without ssl) that causes this error. The zammad ssl-template has also this command at line 59.
</details>

**Comment by MrGeneration** [comment]
This is a technical question, the documentation tells you everything you need to do.

Zammad's installer will not touch your apache moduls for you, you'll have to do this:
https://docs.zammad.org/en/latest/getting-started/configure-webserver.html

**Comment by byPARSE** [comment]
You are right! I know this, but i think the apache2-package-install has to work like the nginx-way: "Out-of-the-box without an error at the beginning". Maybe it is possible to enable the needed mods for apache2 with our package-installer? Its only for the first impression for the admin. In nginx-envs the admin won't get an error at the end of our package-installer.

**Comment by MrGeneration** [comment]
nginx doesn't need additional modules which is why it works out of the box. This is no bug, if it is anything, then a feature request.
As you're supposed to enable ssl and thus have to touch configurations any way, and our documentation does state the next steps, I'd personally say it's an user error to exit the "process" early.

![Image](https://github.com/user-attachments/assets/a2b3e436-2529-449b-be5d-c61bde160fb2)


Have PO decide this if in doubt.

**Comment by YetAnotherGerrit** [comment]
![Image](https://github.com/user-attachments/assets/cba03fc4-613f-4bdc-ba61-e8b40cc42fbc)

We should add a note to `Add your fully qualified [...]` that says something like `Ensure required modules are enabled if apache does not start`.

**Comment by fliebe92** [comment]
It's already part of our [documentation](https://docs.zammad.org/en/latest/getting-started/configure-webserver.html#adjusting-the-webserver-configuration), is this really needed?

**Comment by YetAnotherGerrit** [comment]
> It's already part of our [documentation](https://docs.zammad.org/en/latest/getting-started/configure-webserver.html#adjusting-the-webserver-configuration), is this really needed?

It is part of our documentation, but if you follow our documentation step by step, it comes too late. We could add it to the prerequisite section to avoid users running into the above error message, but I would rather not mention "apache" in our step by step-guide at this point, as we prefer the users to use nginx.

**Comment by fliebe92** [comment]
I cannot follow, sorry:

![Image](https://github.com/user-attachments/assets/314c67d7-9be0-4950-ad77-9357f9380f82)

**Comment by YetAnotherGerrit** [comment]
When you follow our [package installation](https://docs.zammad.org/en/latest/install/package.html) guide "Adjusting the Webserver configuration" is listed on "Next steps". At this point you already experienced above red error message if you installed Zammad on a system with apache2 installed.

So, the knowledge is there, as you've shown, but too late in the process of the guide. We could place the knowledge more prominent and earlier on, but this would mean that we also would promote the usage of apache2 more prominent. Which is something I want to avoid.

**Comment by MrGeneration** [comment]
> So, the knowledge is there, as you've shown, but too late in the process of the guide.

Well we could also stop fiddling and restarting services, no matter if nginx or apache2. When we're at that, we should also rethink if Zammad always should start automatically or if there should be an option to not do that.
